provider/marathon/marathon.go

```diff
@@ -537,3 +676,12 @@ func parseIndex(index string, length int) (int, error) {
 
 	return parsed, nil
 }
+
+func extractServiceName(varg ...string) string {
+	switch {
+	case len(varg) == 1:
+		return varg[0]
+	default:
```

Comment:
You are correct, we did have that established, I just thought it would be nice to still preserve the method signatures to save people who have custom templates the need to change them.  But can remove it if you think that would be better.